.travis.yml

```diff
@@ -1,7 +1,29 @@
+sudo: required
+
 language: node_js
 node_js:
-  - "stable"
+- stable
+
+services:
+- docker
+
+before_install:
+- docker build -t kevinchu/first-blog-mern .
```

Comment:
I think this should probably be in script, but it really depends on what you want to achieve. Check out [travis' build lifecycle](https://docs.travis-ci.com/user/customizing-the-build/#The-Build-Lifecycle) for options.

It shouldn't be in `before_install` since it `install` is about installing dependencies, and therefore `before_install` is about tasks that need to be done before `install`

Reply:
Moved.